initializr-service/application.yml

```diff
@@ -36,6 +36,14 @@ initializr:
         version: 1.0.0.BUILD-SNAPSHOT
         additionalBoms: [cloud-bom]
         repositories: spring-snapshots,spring-milestones
+      scs-bom:
+        groupId: io.pivotal.spring.cloud
+        artifactId: spring-cloud-services-dependencies
+        mappings:
+          - versionRange: "[1.3.0.M2,1.3.5.RELEASE]"
```

Comment:
I guess `1.3.0.RELEASE` would work there. What is the reason you can't support 1.4 ? Spring Cloud does now I think.